tests/rustdoc-json/attrs/must_use.rs

```diff
@@ -1,9 +1,9 @@
 #![no_std]
 
-//@ is "$.index[?(@.name=='example')].attrs" '["#[attr = MustUse]"]'
+//@ is "$.index[?(@.name=='example')].attrs[*].must_use.reason" null
```

Comment:
Does this distinguish between "`must_use` was not present" and "`must_use` was present but with empty reason"?